lib/chopperbot/router.ex

```diff
@@ -61,4 +55,28 @@ defmodule Chopperbot.Router do
   match _ do
     send_resp(conn, 404, "not found")
   end
+
+  defp build_response(input_text) do
+    case Split.run(input_text) do
+      {:ok, ok_msg} ->
+        Character.happy_talk() <> "\n\n" <> ok_msg
+
+      {:error, error_msg} ->
+        Character.confused_talk() <> "\n\n" <> error_msg
+    end
+  end
+
+  defp build_line_suggestion_response do
+    [
+      "Now I can help you split the bill 💸! Just type `split` following by orders. For example...",
+      "",
+      "1️⃣",
+      "split alice 100 alice 250 bob 200 +vat +service",
+      "2️⃣",
+      "split alice 100 bob 200 +v",
+      "3️⃣",
+      "split alice 100 bob 200 share 100"
```

Comment:
Oh, out of the scope, but maybe we can add the discount use case as well. (maybe later)